cmd/nodeslicecontroller/node_slice_controller.go

```diff
@@ -0,0 +1,90 @@
+package main
```

Comment:
the other controller runs per daemonset this will run as its own pod, in the future we can get the other controller to run in the same pod